org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/DrillCommand.java

```diff
@@ -55,11 +55,12 @@ public boolean service(AuRequest request, boolean everError) {
 		String columnName = (String) data.get(0);
 		String tableName = MQuery.getZoomTableName(columnName);
 		String code = (String) data.get(1);
```

Comment:
for _ID column, it is better that code is integer instead of string. if you examine the query object for m_product_id, the where clause is m_product_id='130' instead of m_product_id=130.